region/norfair/east/Single Chamber.json

```diff
@@ -1209,6 +1209,22 @@
       "requires": [],
       "bypassesDoorShell": true
     },
+    {
+      "link": [3, 1],
+      "name": "Carry Grapple Teleport",
```

Comment:
`canGrappleTeleportWallEscape` not worth adding?

Reply:
Good catch, yeah it can be useful for reaching node 4 if you are walljumpless and carrying a flash suit. Similarly with 2->1. Added those now.